app/controllers/status_messages_controller.rb

```diff
@@ -1,128 +1,97 @@
 #   Copyright (c) 2010-2011, Diaspora Inc.  This file is
 #   licensed under the Affero General Public License version 3 or later.  See
 #   the COPYRIGHT file.
-
 class StatusMessagesController < ApplicationController
   before_action :authenticate_user!
 
-  before_action :remove_getting_started, :only => [:create]
+  before_action :remove_getting_started, only: :create
 
-  respond_to :html,
-             :mobile,
-             :json
+  respond_to :html, :mobile, :json
 
-  layout 'application', only: :bookmarklet
+  layout "application", only: :bookmarklet
 
   # Called when a user clicks "Mention" on a profile page
   # @param person_id [Integer] The id of the person to be mentioned
   def new
-    if params[:person_id] && @person = Person.where(:id => params[:person_id]).first
+    if params[:person_id] && get_person(params[:person_id])
```

Comment:
`get_` prefixes are always smelly in Ruby, I'd opt for `fetch_person` or `find_person`.